script/deployParameters/polygon.json

```diff
@@ -3,6 +3,7 @@
   "weth9": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
   "seaport": "0x00000000006c3852cbEf3e08E8dF289169EdE581",
   "seaportV2": "0x00000000000006c7676171937C444f6BDe3D6282",
+  "openseaConduit": "0x0000000000000000000000000000000000000000",
```

Comment:
I'm not sure if the opensea conduit on mainnet is the same on polygon as its not verified on polyscan

Reply:
its not verified but the creating address is the `ConduitController` so I think that must be it

Reply:
yep, confirmed in shared channel

Reply:
Can you add here then